torch/testing/_internal/common_distributed.py

```diff
@@ -111,6 +114,9 @@ def wrapper(*args, **kwargs):
         world_size = int(os.environ["WORLD_SIZE"])
         if torch.cuda.device_count() < world_size:
             sys.exit(TEST_SKIPS[f"multi-gpu-{world_size}"].exit_code)
+        if TEST_HPU:
+            if torch.hpu.device_count < world_size:
```

Comment:
wouldn't it be better to do it in a single line? i.e.
if TEST_HPU and torch.hpu.device_count < world_size: